torch/_inductor/constant_folding.py

```diff
@@ -102,7 +103,8 @@ def run_node(self, node):
             def set_env(arg):
                 self.env[arg] = self.unknown_value
 
-            pytree.tree_map_only(torch.fx.Node, set_env, node.args)
+            # In-place is fine since we don't mutate
+            pytree.tree_map_only_(torch.fx.Node, set_env, node.args)
```

Comment:
nit: unrelated..

Reply:
True (Related only in that I discovered it while looking at the perf of the pattern matcher). I kind of figured for a single-line fix it wasn't worth a PR on its own (especially considering how long it takes to test/review a PR).